src/curve/stable/StablePair.sol

```diff
@@ -127,7 +132,12 @@ contract StablePair is ReservoirPair {
     function burn(address) external override returns (uint256, uint256) {
         // DELEGATE TO StableMintBurn
         address lTarget = MINT_BURN_LOGIC;
-        assembly {
+
+        // SAFETY:
+        // This block is memory safe as as the delegated call has the same signature as the calling function
+        // and both the calldata and returndata do not exceed 64 bytes
+        // This is only valid when lTarget == MINT_BURN_LOGIC
```

Comment:
```suggestion
        // The delegated call has the same signature as the calling function
        // and both the calldata and returndata do not exceed 64 bytes
        // This is only valid when lTarget == MINT_BURN_LOGIC
```

Reply:
fixed this and the block for `mint` as well